lib/syscall_shim/uk_syscall_binary.c

```diff
@@ -75,11 +75,11 @@ void ukplat_syscall_handler(struct __regs *r)
 
 #if CONFIG_LIBSYSCALL_SHIM_HANDLER_ULTLS
 	/* Activate Unikraft TLS */
-	orig_tlsp = ukplat_tlsp_get();
+	ultlsp = ukplat_tlsp_get();
 	self = uk_thread_current();
 	UK_ASSERT(self);
 	ukplat_tlsp_set(self->uktlsp);
-	_uk_syscall_ultlsp = orig_tlsp;
+	_uk_syscall_ultlsp = ultlsp;
```

Comment:
With the current PR the purpose of `ultlsp` is reduced to printing that message below. Personally I wouldn't mind getting rid of that message in favor of less redundancy.

Reply:
Ok, fair enough